lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+#: messages.c:87
+msgid "Resuming print"
+msgstr "Aterupptar utskrift"
+
+# MSG_BED_CORRECTION_RIGHT c=14
+#: ultralcd.cpp:3115
+msgid "Right side[um]"
+msgstr "H\xefger sida[um]"
+
+# MSG_RPI_PORT c=13
+#: messages.c:142
+msgid "RPi port"
+msgstr "RPi port"
+
+# MSG_WIZARD_RERUN c=20 r=7
+#: ultralcd.cpp:4756
+msgid "Running Wizard will delete current calibration results and start from the beginning. Continue?"
+msgstr "Att utf\xefra guiden kommer att radera aktuella kalibreringsinst\xe1llningar och b\xefrja om fran b\xefrjan. Forts\xe1tta?"
+
+# MSG_SD_CARD c=8
+#: messages.c:138
+msgid "SD card"
+msgstr "SD-kort"
+
+# MSG_RIGHT c=10
+#: ultralcd.cpp:2845
+msgid "Right"
+msgstr "H\xefger"
+
+# MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=20 r=3
+#: messages.c:44
+msgid "Searching bed calibration point"
+msgstr "S\xefker efter kalibreringspunkt f\xefr b\xe1dden"
+
+# MSG_LANGUAGE_SELECT c=18
+#: ultralcd.cpp:4451
+msgid "Select language"
+msgstr "V\xe1lj sprak"
+
+# MSG_SELFTEST_OK c=20
+#: ultralcd.cpp:7679
+msgid "Self test OK"
+msgstr "Sj\xe1lvtest OK"
+
+# MSG_SELFTEST_START c=20
+#: ultralcd.cpp:7447
+msgid "Self test start"
+msgstr "Sj\xe1lvteststart"
+
+# MSG_SELFTEST c=18
+#: ultralcd.cpp:5780
+msgid "Selftest"
+msgstr "Sj\xe1lvtest"
+
+# MSG_SELFTEST_ERROR c=20
+#: ultralcd.cpp:8121
+msgid "Selftest error!"
+msgstr "Sj\xe1lvtestfel!"
+
+# MSG_SELFTEST_FAILED c=20
+#: messages.c:90
+msgid "Selftest failed"
+msgstr "Sj\xe1lvtestet felade"
+
+# MSG_FORCE_SELFTEST c=20 r=8
+#: Marlin_main.cpp:1637
+msgid "Selftest will be run to calibrate accurate sensorless rehoming."
+msgstr "Sj\xe1lvtest kommer att utf\xefras f\xefr att kalibrera exakt sensorl\xefs hemposition."
+
+# MSG_SEL_PREHEAT_TEMP c=20 r=6
+#: ultralcd.cpp:4998
+msgid "Select nozzle preheat temperature which matches your material."
+msgstr "V\xe1lj munstycksf\xefrv\xe1rmningstemperatur som passar ditt material."
+
+# MSG_SET_TEMPERATURE c=19
+#: ultralcd.cpp:3135
+msgid "Set temperature:"
+msgstr "S\xe1tt temperatur:"
+
+# MSG_SETTINGS c=18
+#: messages.c:99
+msgid "Settings"
+msgstr "Inst\xe1llningar"
+
+# MSG_SHOW_END_STOPS c=18
+#: ultralcd.cpp:5801
+msgid "Show end stops"
+msgstr "Visa \xe1ndl\xe1gen"
+
+# MSG_FILE_CNT c=20 r=6
+#: cardreader.cpp:817
+msgid "Some files will not be sorted. Max. No. of files in 1 folder for sorting is 100."
+msgstr "Vissa filer kommer inte att sorteras. Max. antal filer i 1 mapp f\xefr sortering \xe1r 100."
+
+# MSG_SORT c=7
+#: messages.c:139
+msgid "Sort"
+msgstr "Sortera"
+
+# MSG_NONE c=8
+#: messages.c:129
+msgid "None"
+msgstr "Ingen"
+
+# MSG_SORT_TIME c=8
+#: messages.c:140
+msgid "Time"
+msgstr "Tid"
+
+# MSG_SEVERE_SKEW c=14
+#: ultralcd.cpp:2888
+msgid "Severe skew"
+msgstr "H\xefg skevhet"
+
+# MSG_SORT_ALPHA c=8
+#: messages.c:141
+msgid "Alphabet"
+msgstr "Alfabet"
+
+# MSG_SORTING c=20
+#: cardreader.cpp:880
+msgid "Sorting files"
+msgstr "Sorterar filer"
+
+# MSG_SOUND_LOUD c=7
+#: messages.c:144
+msgid "Loud"
+msgstr "H\xe1gt"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.